### PR TITLE
[BACKLOG-34482] Backport of ORC/Parquet support for ADLS2 VFS

### PR DESCRIPTION
**Pull request by renjith-nem**
Master Branch PR's:

- https://github.com/pentaho/pentaho-hadoop-shims/pull/1188

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 3m 37s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl shims/apache/driver
```
---
:ok_hand: All tests passed!
 
Tests run: **35**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/big-data-hadoop-shims/job/PR-1187/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 3m 8s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl shims/apache/driver
```
---
:ok_hand: All tests passed!
 
Tests run: **35**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/big-data-hadoop-shims/job/PR-1187/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by lucboudreau — APPROVED** [review]

File: shims/apache/driver/src/test/java/org/pentaho/hadoop/shim/pvfs/conf/ADLSGen2ConfTest.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
+package org.pentaho.hadoop.shim.pvfs.conf;
```
Comment (ddiroma) [inline shims/apache/driver/src/test/java/org/pentaho/hadoop/shim/pvfs/conf/ADLSGen2ConfTest.java:1]: @renjith-nem This is a test file. I think it still needs copyright.

**Review by ddiroma — APPROVED** [review]

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 3m 3s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl shims/apache/driver
```
---
:ok_hand: All tests passed!
 
Tests run: **35**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/big-data-hadoop-shims/job/PR-1187/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by renjith-nem** [comment]
@ssamora We would need to backport the new changes to the 9.1 branch.
I've linked the master branch PR's in the description.

Thank you !!

**Review by ssamora — APPROVED** [review]

**Pull request merged**